.github/workflows/build.yml

```diff
@@ -59,16 +59,16 @@ jobs:
         run: python -m cibuildwheel --archs native --output-dir ../dist
       - uses: actions/upload-artifact@v4
         with:
-          name: wheels-${{ matrix.os }}
+          name: wheels-${{ runner.os }}
```

Comment:
This is actually less safe -- if we eventually add `macos-14` i.e. arm64 wheels, the name for both will be `wheels-macOS` instead of being separate `wheels-macos-13` and `wheels-macos-14`

Reply:
I just dropped testing macos-12-- couldn't think of a nice way to have that runner pull from macos-13 for wheels?

It looks like I used to dump all the wheels in the same archive. The solution in that case would be to preinstall numpy, then `pip install --noindex --find-links <path> psychtoolbox`, and then pip would figure out the right one?